DEPLOYMENT_CHECKLIST_v2.3.md

```
@@ -0,0 +1,405 @@
+# Wait for postgres to be ready
+sleep 10
+
+# Run migration
+docker exec -i sms_postgres psql -U postgres -d sms_bridge < init/migration_v2.3_monitoring.sql
+
+# Verify migration
+docker exec sms_postgres psql -U postgres -d sms_bridge -c "\dt monitoring_*"
+```
+
+Expected output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)